fblive.php

```diff
@@ -9,13 +9,25 @@
 $shoutouts = include __DIR__ . '/shoutouts.php';
 $settings = include __DIR__ . '/settings.php';
 
+/*
+ * Ensure the correct settings are set in order to run the script
+ */
+if(!isset($settings['ACCESS_TOKEN']) || empty($settings['ACCESS_TOKEN'])) {
```

Comment:
```php
if (empty($settings['ACCESS_TOKEN'])) {
```

actually covers both of these cases in PHP. `empty` turns off error reporting, which means it also checks if the variable is set.